content/docs/started/getting-started-k8s.md

```diff
@@ -4,84 +4,156 @@ description = "Instructions for installing Kubeflow on your existing Kubernetes
+GCP_ZONE="us-west1-b"
+
+CLUSTER_VERSION="$(gcloud container get-server-config --format="value(validMasterVersions[0])")" 
+CLUSTER_NAME="kubeflow"
 
-1. Download a `kfctl` release from the 
-  [Kubeflow releases page](https://github.com/kubeflow/kubeflow/releases/).
+############################
+# Create and setup cluster #
+############################
 
-1. Unpack the tar ball:
+gcloud container clusters create ${CLUSTER_NAME} \
+--project ${GCP_PROJECT} \
+--zone ${GCP_ZONE} \
+--cluster-version ${CLUSTER_VERSION} \
+--machine-type "n1-standard-8" --num-nodes "1" \
+--image-type "UBUNTU" \
+--disk-type "pd-ssd" --disk-size "50" \
+--no-enable-cloud-logging --no-enable-cloud-monitoring \
+--no-enable-ip-alias \
+--enable-autoupgrade --enable-autorepair
+
+echo "Getting credentials for newly created cluster..."
+gcloud container clusters get-credentials ${CLUSTER_NAME} --zone=${GCP_ZONE}
+
+echo "Setting up GKE RBAC..."
+kubectl create clusterrolebinding cluster-admin-binding --clusterrole=cluster-admin --user=${GCP_USER}
+```
+
+</details>
+
+### Deploy Kubeflow
+
+Follow these steps to deploy Kubeflow:
+
+1. Download a `kfctl` release from the [Kubeflow releases page](https://github.com/kubeflow/kubeflow/releases/) and unpack it:
 
     ```
     tar -xvf kfctl_<release tag>_<platform>.tar.gz
     ```
 
-1. Run the following commands to set up and deploy Kubeflow. The code below
-  includes an optional command to add the binary `kfctl` to your path. If you 
-  don't add the binary to your path, you must use the full path to the `kfctl` 
-  binary each time you run it.
-
-    ```bash
-    # The following command is optional, to make kfctl binary easier to use.
-    export PATH=$PATH:<path to kfctl in your kubeflow installation>
-
-    export KFAPP=<your choice of application directory name>
-    # Default uses IAP.
-    kfctl init ${KFAPP}
-    cd ${KFAPP}
-    kfctl generate all -V
-    kfctl apply all -V
+   Alternatively, you can easily build the `kfctl` binary yourself:
+
     ```
-   * **${KFAPP}** - the _name_ of a directory where you want Kubeflow 
-     configurations to be stored. This directory is created when you run
-     `kfctl init`. If you want a custom deployment name, specify that name here.
-     The value of this variable becomes the name of your deployment.
-     The value of this variable cannot be greater than 25 characters. It must
-     contain just the directory name, not the full path to the directory.
-     The content of this directory is described in the next section.
+    git clone https://github.com/kubeflow/kubeflow.git
+    cd kubeflow/bootstrap
+    make build-kfctl-container
+    ```    
+    
+2. Run the following commands to set up and deploy Kubeflow. The code below includes an optional command to add the binary `kfctl` to your path. If you don't add the binary to your path, you must use the full path to the `kfctl` binary each time you run it.
```

Comment:
The list formatting is broken in the rendered view - see preview: https://deploy-preview-861--competent-brattain-de2d6d.netlify.com/docs/started/getting-started-k8s/

To fix this, you should indent the preceding paragraphs so that the renderer includes them in the previous bullet point.

In the Markdown for numbered lists, please use a sequence of `1.`s rather than `1.` and `2.` etc. The former is easier to maintain when we want to add/remove bullet points. The rendered result is the same in both cases. Example:

```
1. First item.
1. Second item.
1. Third item.
```
The above yields this list:
1. First item.
1. Second item.
1. Third item.

Reply:
Thanks for catching that and helping me solve it, I will fix it asap.